elements/pl-number-input/pl-number-input.css

```diff
@@ -2,3 +2,14 @@
     width: 400px;
     max-width: 100%;
 }
+
+.pl-number-input-inline-badge {
+    position: relative;
+    top: 9px;
+    height: 17px;
+}
+
+.pl-number-input-force-radius-border {
+    border-top-right-radius: 0.25em !important;
+    border-bottom-right-radius: 0.25em !important;
+}
```

Comment:
These two CSS styles (`pl-number-input-inline-badge` and `pl-number-input-force-radius-border`) seem really specific, and brittle with respect to things like future bootstrap updates. Are they necessary?